packages/react/src/themes/teams/componentVariables.ts

```diff
@@ -63,6 +63,8 @@ export { default as Status } from './components/Status/statusVariables'
 
 export { default as Text } from './components/Text/textVariables'
 
+export { default as Toolbar } from './components/Toolbar/toolbarVariables'
```

Comment:
Can we export these for the other Toolbar components as well (as we are exporting the menuVariables for the MenuItem too)?